src/client/Microsoft.Identity.Client/Internal/Broker/NullBroker.cs

```diff
@@ -80,5 +81,10 @@ public Task<MsalTokenResponse> AcquireTokenByUsernamePasswordAsync(Authenticatio
             _logger.Info("NullBroker - returning null on ROPC request.");
             return Task.FromResult<MsalTokenResponse>(null);
         }
+
+        public IReadOnlyDictionary<string, string> GetSsoPolicyHeaders()
```

Comment:
nit: It's more performant to iterate over concrete types rather than the interfaces.
```suggestion
        public ReadOnlyDictionary<string, string> GetSsoPolicyHeaders()
```